build/configs/artik05x/artik05x_download.sh

```diff
@@ -67,10 +69,27 @@ Options:
 EOF
 }
 
+prepare_resource()
+{
+    if [ -d "${RESOURCE_DIR_PATH}" ]; then
+        # When CONGIG_FRAME_POINTER flag is enabled, copy the build/output/bin/System.map file
+        # to tools/fs/contents directory so that symbol names can be displayed using dump_stack
+        if [ "${CONFIG_FRAME_POINTER}" == "y" ]; then
+            echo "Copying System.map file from ${OUTPUT_BINARY_PATH} to ${RESOURCE_DIR_PATH}"
```

Comment:
We guide two methods to download binary. But this is valid only on "make download". How about move this functionality to post work of Makefile.xx? I think that is better.